src/lib/output/themes/default/assets/typedoc/components/Accordion.ts

```diff
@@ -25,7 +29,16 @@ class AccordionImpl {
         for (const acc of this.accordions) {
             acc.open = open;
         }
-        storage.setItem(this.key, open.toString());
+        if (!this.disableStorage) {
+            storage.setItem(this.key, open.toString());
+        }
```

Comment:
I'm really not a fan of how this option has propagated to **every** instance of saving something to storage. A better implementation would instead be to have the storage module perform the correct action based on whether the option was set or not, so this option doesn't leak into every instance of saving state.